Livecheckables/musepack.rb

```diff
@@ -0,0 +1,6 @@
+class Musepack
+  livecheck do
+    url "https://www.musepack.net/index.php?pg=src"
+    regex(/href=.*?musepack[._-]src[._-](r\d+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?musepack(?:[._-]src)?[._-](r\d+)\.t/i)
```

Seeing as the other source files don't include `_src` in the filename, it probably wouldn't be a bad idea to allow matching of a file name like `musepack_r475.tar.gz` as well.